src/main/java/com/dnsimple/request/CertificateRenewalPurchaseOptions.java

```diff
@@ -0,0 +1,32 @@
+package com.dnsimple.request;
+
+public class CertificateRenewalPurchaseOptions {
```

Comment:
I wanted to suggest we rename this to `CertificateRenewalOptions` for brevity if that doesn't clash with the naming scheme we're using in other clients

Reply:
I also think it's a bit verbose but the `purchaseLetsencryptCertificate` method takes `CertificatePurchaseOptions` so I feel `CertificateRenewalPurchaseOptions` is more consistent. Alternatively we could drop `Purchase` from both names?

I had a look at some other clients and we do use `Purchase` and `PurchaseRenewal` (not `Renewal`):
- Python: `LetsencryptCertificateInput` and `LetsencryptCertificateRenewalInput`
- Rust: `LetsEncryptPurchasePayload` and `LetsEncryptPurchaseRenewalPayload`
- Go: `LetsencryptCertificateAttributes` for both APIs